components/courses/spec/features/admin/lectures/create_spec.rb

```diff
@@ -17,17 +17,28 @@
 
       expect_an_error lecture_title: :blank
     end
+
+    it 'validates errors when time is wrong' do
+      allow_any_instance_of(Courses::Lecture::TimePolicy).to receive(:allowed?).and_return(false)
```

Comment:
I would prefer to choose invalid date here and check if it is really validated.
Feature test purpose is to test all system together and not to mock/stub anything.
That is my opinion.